File: PAT (Basic-Level)/C/1023.c
#include <stdio.h>

int main()
{
	int i;
	int num_rec[10];
	for(i = 0; i < 10; i++){
		scanf("%d", &num_rec[i]);
	}
/*	for(i = 0; i < 10; i++){
		printf("%d ", num_rec[i]);
	}*/
	i = 0;
	while(i < 9){
		i++;
		if(num_rec[i] != 0){
			printf("%d", i);
			num_rec[i]--;
			break;
		}
	}
	i = -1;
	while(i < 9){
		i++;
		while(num_rec[i] != 0){
			printf("%d", i);
			num_rec[i]--;
		}
	}
	return 0;
}

File: PAT (Basic-Level)/Python/1023.py
#!/usr/bin/python3
# -*- coding: utf-8 -*-

num_rec = list(map(int, input().split()))
i = 0
while i < 9:
    i += 1
    if num_rec[i] != 0:
        print(i, end = '')
        num_rec[i] -= 1
        break
i = -1
while i < 9:
    i += 1
    while num_rec[i] != 0:
        print(i, end = '')
        num_rec[i] -= 1